File: Assignment_No-36/Question_01.c
/*1)write a java program which accept number from user and return the count even digit

input:2395

output:1

*/

//////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.*;

class Digits
{
    public void CountEvenDigits(int iNo)
    {
        int iCount = 0;
        int iDigit = 0;

        while(iNo != 0)
        {
            iDigit = iNo % 10;
            if(iDigit%2==0)
            {
                iCount++;
            }
            iNo = iNo / 10;
        }
        System.out.println("Frequency of Even  is : "+iCount);
    }
}

class program186
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int iValue = 0;
        
        System.out.println("Enter number : ");
        iValue = sobj.nextInt();

        Digits dobj = new Digits();
        dobj.CountEvenDigits(iValue);

       
    }
}

File: Assignment_No-37/Question_01.java
/*1)write java program which accept N numbers from user  and accept one another number as NO,check whether number 
No is present or not.
input:N=6
No=66
elements:85 66 3 66 93 88
output=true

input:N=6
No=12
elements:85 66 3 66 93 88
output=False

*/

////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.*;
class Number
{
   public int Arr[];
	public Number(int iSize)
	{
		 Arr=new int[iSize];

    }
    public void Accept()
    {
    	int i=0;
    	Scanner sobj=new Scanner(System.in);
    	System.out.println("Enter the number of elments:");

    	for(i=0;i<Arr.length;i++)
    	{
    		Arr[i]=sobj.nextInt();
    	}
    }
    public void Display()
    {
        int i = 0;
        System.out.println("Elements of the array are : ");

        for(i = 0 ; i < Arr.length; i++)
        {
            System.out.println(Arr[i]);
        }
    }
    public boolean Check(int iNo)
    {

         boolean dRet=false;
    	int i=0;
    	for(i=0;i<Arr.length;i++)
    	{
    		if(Arr[i]==iNo)
    		{
    		     dRet=true;
    		}

    	}
    	return dRet;
    }
}
class ass191
{
	public static void main(String Arg[])
	{

        int iLength=0;
        int iSearch=0;
        boolean bRet=false;
		Scanner sobj=new Scanner(System.in);


		System.out.println("Enter the number:");
		iLength=sobj.nextInt();

		System.out.println("Enter the number you want to search:");
		iSearch=sobj.nextInt();

		Number nobj=new Number(iLength);
		nobj.Accept();
		nobj.Display();
		bRet=nobj.Check(iSearch);


     if(bRet==true)
     {
     	System.out.println("Number is presnet");
     }
     else
     {

     	System.out.println("Number is not presnet");

     }
} 
}
